eslint/react/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
   },
   extends: ['plugin:react/recommended'],
-  plugins: ['react'],
+  plugins: ['react', '@babel/plugin-syntax-jsx'],
```

Comment:
This is interesting! I’m kind of surprised this works, passing a Babel plugin as an ESLint plugin.

Is there some documentation around on doing this?

Reply:
Honestly, I thought it was a bit weird too but looking at the error message made me think it must 'work' (above screenshot kinda cut out the eslint msg), but below: 

![Screenshot 2021-05-13 at 15 05 40](https://user-images.githubusercontent.com/17278432/118137126-c13c0a00-b3fc-11eb-9188-e3d92cb334ec.png)

When I tested doing that in the local version it worked, I then tried it on the react ling config by doing a `yarn link` to ensure nothing else broke.

I went digging for docs around this but came up short, the link in the error msg is just a link to the plugin itself (https://git.io/vb4yA).

Reply:
I think it's the plugins section of the _babel_ config that needs this added